airswap/airswap-marketplace
# 62 update balances store to support individual tokens

# Description

Fixes #62

Fetching Token Ids using ERC165 to check which interface is supported:
- if ERC721Enumerable -> using balanceOf and tokenOfOwnerByIndex methods
- if ERC721 -> getting token ids based on transfer events from address 0x00 to wallet address
- if ERC1155 -> getting token ids based on transfer events from address 0x00 to wallet address

# Steps to test the change

1.  Make sure to have ERC721Enumerable, ERC721 and ERC1155 NFTs in your wallet
2. Set the collection address in `.env` file. 
3. Verify balance stop is updated with the token ids of the NFTs you own4. 
# Checklist:
